lexicons/app/bsky/actor/defs.json

```diff
@@ -47,6 +48,7 @@
           "ref": "#profileAssociated"
         },
         "indexedAt": { "type": "string", "format": "datetime" },
+        "createdAt": { "type": "string", "format": "datetime" },
```

Comment:
This relates to some things we want to do specifically for newskies.  It relates to starter packs because we may want to feature the starter pack someone used to join on their profile, but only for a period of time.

We'll probably give `createdAt` the same amount of credence we do for posts or the many other records that have it, and lean more on the server to determine when the user started using Bluesky.  But for completeness I figured it should probably be reflected on repo in some way (e.g. for the next appview's benefit!).  Most records have the `createdAt` field, so it felt natural enough to me to include it.